src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -212,6 +212,13 @@ void VtolAttitudeControl::vehicle_cmd_poll()
 				uORB::Publication<vehicle_command_ack_s> command_ack_pub{ORB_ID(vehicle_command_ack)};
 				command_ack_pub.publish(command_ack);
 			}
+
+		} else if (_vtol_vehicle_status.vtol_transition_failsafe &&
+			   (vehicle_command.command == vehicle_command_s::VEHICLE_CMD_DO_SET_MODE ||
+			    vehicle_command.command == vehicle_command_s::VEHICLE_CMD_DO_REPOSITION)) {
+			// Exit failsafe state if an external command arrives
+			_vtol_vehicle_status.vtol_transition_failsafe = false;
```

Comment:
Hi @Igor-Misic 
I think this looks like a good, clean place to solve the problem.
I have a small concern if an external command is received about the same time as the QuadChute happening.
If the failure flag is cleared the first iteration after it is set, I think these lines
https://github.com/aviant-tech/PX4-Autopilot/blob/e24e20a4958f6ccd129463c9cad8eb57a9d4f8e9/src/modules/vtol_att_control/standard.cpp#L110-L120
may not be executed.
The chance if this being an issue is very small, but perhaps it should be addressed.

What do you think about just removing `_vtol_vehicle_status.vtol_transition_failsafe = false;` from your code, and let existing code reset the failure flag based on `_transition_command`? This way the behaviour would also be more similar to toggling the MR/FW switch on a remote(?).

Reply:
I was in doubt about doing it that way also. Thanks for your feedback, I have changed it now.